components/feature/contextmenu/src/main/java/mozilla/components/feature/contextmenu/DefaultSelectionActionDelegate.kt

```diff
@@ -68,28 +69,31 @@ class DefaultSelectionActionDelegate(
         else -> null
     }
 
-    override fun performAction(id: String, selectedText: String): Boolean = when (id) {
-        SEARCH -> {
-            searchAdapter.sendSearch(false, selectedText)
-            true
+    override fun performAction(id: String, selectedText: String): Boolean {
+        trackContextMenuOption?.invoke(id)
```

Comment:
Just a drive-by: Please add a unit test to `DefaultSelectionActionDelegateTest` to make sure we're invoking the lambda.

Reply:
Hey, thanks for taking a look. I've switched to emitFact method that @jonalmeida suggested.